Dreamscapes/Ldap/Core/Ldap.php

```diff
@@ -752,11 +757,10 @@ public function __call($method, $args)
         return call_user_func_array([$this, 'ldapSearch'], $args);
     }
 
-
     /**
      * Check the ldap status code and throw exception on error
      *
-     * @return void
```

Comment:
But it does not return anything... How do document a function that does not return anything? Also, for consistency and for being explicit, I prefer to include a `@return` docblock for all methods.